### ISSUE TITLE
Accessibility problems with server, accordion

### ISSUE THREAD

**Issue by kevee** [swagger-api/swagger-ui#6015]
### Q&A (please complete the following information)
 - OS: macOS
 - Browser: chrome
 - Version: [e.g. 22]
 - Method of installation: npm
 - Swagger-UI version: 3.25
 - Swagger/OpenAPI version: OpenAPI 3.0

### Content & configuration
Sample page: https://covidtracking.com/api
JSON: http://covidtracking.com/api-docs/COVID-tracking-endpoints-1.0-docs.json

### Describe the bug you're encountering
There are two major accessibility problems with Swagger UI. We are tracking these issues in our own repo, but I'm reproducing those comments below:

### Server dropdown is missing a label 
[Original issue](https://github.com/COVID19Tracking/website/issues/866)
#### Element
[COVID Tracking API Sandbox](https://covidtracking.com/api#status)
Note the above link actually goes to the element above it; this element didn't have an anchor link.
Maybe it should? Separate issue.

<img width="304" alt="select input with https://covidtracking.com and server text above it. Server text is not the label" src="https://user-images.githubusercontent.com/7484549/82087456-fce6ab80-96a4-11ea-8af1-bccb0a8cd6de.png">

#### Expected Behaviour
Any form element should have a label so that it is announced with screenreaders. Before any form element, like this dropdown, I'd expect for the screen reader to announce its label.

#### Actual Behaviour
The screen reader announces that I'm about to go into a popup button and reads the field, which is the URL. I have no context for what it is unless I can visually see the faux label or I navigate back. This prevents users who are tabbing to interactive elements to know what this is.

#### Recommendation
Move the visual `<span>` that is currently outside of the `<label>` element inside of the `<label>` element so it's properly announced. The `<label>` also shouldn't nest the `<select>` element inside of it. It should read:

```
<label>
</label>
<select>
...
```

#### Resources
[WebAIM: Form controls
](https://webaim.org/techniques/forms/controls)

#### Additional Note
If there is only one server, this element should probably not be rendered at all.

### Accordion content is inaccessible with keyboard
[Original issue](https://github.com/COVID19Tracking/website/issues/865)

#### Element
Any dropdown in the list of endpoints

<img width="1182" alt="Dropdown activated with several GET button-looking things that are not tabable" src="https://user-images.githubusercontent.com/7484549/82085521-0f131a80-96a2-11ea-93c3-1c5882dcc190.png">

#### Expected Behaviour
The ability to tab to the GET elements by keyboard.

#### Actual Behaviour
Each of these GET elements is not tabbable. I can only access them by navigating by mouse.

#### Recommendation
Make all get elements buttons if possible. If it can be a button, that would be best, as it'll bake in all the button goodness.

If it must be a div for some reason, it needs to inherit a role="button" and also get a lot of aria properties to make it function like a button. 

#### Resources
[MDN's button role notes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role).
